src/storage/redis_metadata.cc

```diff
@@ -372,29 +372,31 @@ rocksdb::Status StreamMetadata::Decode(Slice *input) {
     return s;
   }
 
-  if (input->size() < 8 * 11) {
-    return rocksdb::Status::InvalidArgument(kErrMetadataTooShort);
-  }
+  if (Type() == kRedisStream) {
```

Comment:
It is already assumed that in `StreamMetadata::Decode` the type must be kRedisStream, which is checked in other places. So I don't think it is meaningful.